fact-bounty-flask/requirements.txt

```diff
@@ -43,7 +43,7 @@ SQLAlchemy==1.2.18
 tornado==6.0
 tweepy==3.8.0
 typed-ast==1.2.0
-urllib3==1.24.1
+urllib3==1.23.0
```

Comment:
Why is this package version changed??

Reply:
this was changed as a result of a warning coming during the setup.